experimental/traffic-portal/src/app/core/statuses/status-details/status-details.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { StatusDetailsComponent } from './status-details.component';
+
+describe('StatusDetailsComponent', () => {
+  let component: StatusDetailsComponent;
+  let fixture: ComponentFixture<StatusDetailsComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ StatusDetailsComponent ]
+    })
+    .compileComponents();
+
+    fixture = TestBed.createComponent(StatusDetailsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
This should include more than just the default generated test to make sure it can be instantiated. Personally I'm trying to get 100% coverage in my new files, but I wouldn't ask that much of you - although you are welcome to it if you want. Just try to cover as much as you can, but if something's complicated and would take a lot of time to work out you can skip it, stuff like testing route parameter subscriptions and whatnot can get pretty messy pretty quickly.